scripts/install.js

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env node
+"use strict";
+
+const fs = require("fs");
+const path = require("path");
+
+let rootDir = __dirname.split("/node_modules", 1)[0];
+if (rootDir === __dirname) {
+  rootDir = path.resolve(".");
+}
+const pkg = require(path.resolve(rootDir, "package.json"));
+
+// Do not run install script if this package is the root
+if (rootDir !== path.resolve(".")) {
+  fs.writeFileSync(
+    path.resolve(rootDir, "package.json"),
+    JSON.stringify(
+      Object.assign(pkg, {
+        scripts: Object.assign(
+          {
+            format:
+              "prettier --print-width 100 --write *.{js,jsx} `find . -type d -d 1 -exec echo '{}/**/*.{js,jsx}' \\; | egrep -v '(/node_modules/|/dist/|/coverage/)'`",
```

Comment:
@jchip refactored in https://github.com/walmartlabs/wml-coding-std/pull/1/commits/61942fdde99ceb82c583fe5256fc7e3bbadc25b2